### PR TITLE
[SRVKS-743] Disable monitoring if istio is enabled

### PR DESCRIPTION
**Pull request by skonto**
This is intended for 1.16 and should be fixed in 1.17.

### CODE REVIEW

File: openshift-knative-operator/pkg/serving/extension.go [hunk 1]
```diff
@@ -116,6 +116,11 @@ func (e *extension) Reconcile(ctx context.Context, comp v1alpha1.KComponent) err
 			Type: "ConfigMap",
 		}
 	}
+
+	// Temporary fix for SRVKS-743
+	if ks.Spec.Ingress.Istio.Enabled {
+		common.Configure(&ks.Spec.CommonSpec, monitoring.ObservabilityCMName, monitoring.ObservabilityBackendKey, "none")
```
Comment (markusthoemmes) [inline openshift-knative-operator/pkg/serving/extension.go:122]: let's make this `ConfigureIfUnset` just to leave the gate open for people who **really** want to play with it enabled explicitly.

Comment (skonto) [reply openshift-knative-operator/pkg/serving/extension.go:122]: Ok good to know updating in a sec.

**Review by markusthoemmes — APPROVED** [review]
/lgtm
/approve

**Pull request merged**